### DISCUSSION TITLE
cut mp3 silence offset automatically in editor

### DISCUSSION THREAD

**Discussion by h3llo-wor1d** [ppy/osu#19423] [Ideas]
Did some research, would be relatively easy to do. I think it would be cool if the game just automatically did it when you import mp3 files. All mp3s have different offsets which is really annoying, but there is always a specific amount of silence before audio playback starts that fucks up beatmaps, and finding it manually is a pain in the ass.

I may just do a pull request for this, I'm not sure yet. I forget if Lazer is C# or not, but if it is, I definitely will. Anyway, I hope I'm not the first person to ask for it, because this seems like a no-brainer to me.

**Comment by NotGumballer91** [comment]
Lazer is indeed C#
